frontend/src/Layout/Root.jsx

```
@@ -0,0 +1,16 @@
+import { Outlet } from "react-router-dom";
+import Navbar from "../Component/Share/Navbar/Navbar";
+import Footer from "../Component/Share/Footer/Footer";
+
+
+const Root = () => {
+    return (
+        <div>
+            <Navbar></Navbar>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)